src/components/chat/Messages.tsx

```diff
@@ -65,5 +65,5 @@ export default function Messages() {
 }
 
 const List = tw.ol`
-  flex flex-col justify-end h-full overflow-y-scroll
+  flex flex-col h-full overflow-auto
```

Comment:
```suggestion
  flex flex-col h-full mt-auto overflow-y-scroll
```

Removing `justify-end` does fix the scroll problem, thanks for figuring that out! However this also causes messages to appear at the top of the list rather than the bottom. To fix this I've added `mt-auto` and now everything works as intended!

It's a tad nit-picky but since this is a Discord clone, Discord always has a scrollbar for the messages, so I set it back to `overflow-y-scroll`.

Thanks for contributing!